src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -35,15 +37,34 @@ public JsonResult execute() {
         String studentEmail = getNonNullRequestParamValue(Const.ParamsNames.STUDENT_EMAIL);
         // Skip rigorous validations to avoid incurring extra db reads and to keep the endpoint light
 
-        // Necessary to assist local testing. For production usage, this will be a no-op.
-        logsProcessor.createFeedbackSessionLog(courseId, studentEmail, fsName, fslType);
-
         FeedbackSessionAuditLogDetails details = new FeedbackSessionAuditLogDetails();
         details.setCourseId(courseId);
         details.setFeedbackSessionName(fsName);
         details.setStudentEmail(studentEmail);
         details.setAccessType(fslType);
 
+        if (isCourseMigrated(courseId)) {
+            Student student = sqlLogic.getStudentForEmail(courseId, studentEmail);
+            FeedbackSession feedbackSession = sqlLogic.getFeedbackSession(fsName, courseId);
```

Comment:
Inorder to log the student and session ids it would incur additional reads on the creation of every log in GCP.

One idea around this is we can update other APIs to pass student/session ids to the frontend so this API can get the ids from the params?

Reply:
yes, let's do that!